article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -0,0 +1,174 @@
+@import layout.ContentWidths.MainMedia
+@import model.content.MediaAtom
+@import common.Localisation
+@(articleModel: ArticlePage, recipes: Seq[model.content.RecipeAtom])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
+@import common.LinkTo
+@import views.BodyCleaner
+
+@schemaType(page: ArticlePage) = @{ page.article.metadata.schemaType }
+@bodyType(page: ArticlePage) = @{ "articleBody" }
+
+@defining(articleModel.article) { article =>
+    <article id="article" data-test-id="article-root"
+            class="content content--recipes"
+            itemscope itemtype="@schemaType(articleModel)" role="main" >
+    <meta itemprop="mainEntityOfPage" content="@LinkTo(article.metadata.url)">
+    <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
+        <meta itemprop="name" content="The Guardian">
+        @fragments.logo(amp = false)
+    </div>
+
+    <div class="recipes__main">
+        <div class="recipe__images-wrapper">
+            <div class="recipe__label">
+                <h2>food &amp; drink</h2>
+            </div>
+            <div class="recipe__image__wrapper recipe__image__wrapper--is-displayed">
+            @fragments.image(picture = defaultImage,
+                classes = Seq("recipe__image", "fadeIn"),
+                imageAltText = "main alt text here",
+                widths = MainMedia.immersive,
+                isImmersiveMainMedia = true)
+            </div>
+
+            @for((recipe, i) <- recipes.zipWithIndex){
+                <div class="recipe__image__wrapper recipe recipe__@i">
+                @fragments.image(picture = getPicture(recipe),
+                    classes = Seq("recipe__image", "fadeIn"),
+                    imageAltText = "alt text here",
+                    widths = MainMedia.immersive,
+                    isImmersiveMainMedia = true)
+                </div>
+            }
+        </div>
+
+        <div class="recipe__article-wrapper">
+            <div class="recipe__article">
+                <div class="recipe__article--meta">
+                        @articleModel.item.content.blogOrSeriesTag.map { series =>
+                        <div class="content__series-label @if(articleModel.item.content.isImmersive && articleModel.item.content.tags.isArticle) { content__series-label--immersive-article }">
```

Comment:
I don't think you need these if statements here. Just put the classnames you need. You don't have to worry about whether this is an immersive or not, because the styling for this template won't change either way.